azkaban-common/src/main/java/azkaban/DispatchMethod.java

```diff
@@ -22,10 +22,19 @@
  * This enum contains list of dispatch types implemented in Azkaban.
  */
 public enum DispatchMethod {
-  PUSH,
-  POLL,
-  CONTAINERIZED;
+  PUSH(0),
+  POLL(1),
+  CONTAINERIZED(2);
   private static final Logger logger = LoggerFactory.getLogger(DispatchMethod.class);
+  private final int numVal;
+
+  DispatchMethod(final int numVal) {
+    this.numVal = numVal;
+  }
+
+  public int getNumVal() {
```

Comment:
Is there any need to add method which will return enum for given num value?
What is the significance of this num value? Can you please add it in comment?